### ISSUE TITLE
ALTER USER not compatible with MariaDB 10.1 from Debian Stretch

### ISSUE THREAD

**Issue by larsen0815** [webyog/sqlyog-community#2221]
When I try to change a user password using the User Manager for a MariaDB 10.1 server, I get a syntax error:
````
ALTER USER 'testtesttest'@'%' IDENTIFIED BY '456789123'

You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'USER 'testtesttest'@'%' IDENTIFIED BY '456789123'' at line 1
````
ALTER USER is available starting with MariaDB 10.2.0 (see https://mariadb.com/kb/en/library/alter-user/). Therefore, all current Debian releases of MariaDB (jessie, stretch) will be unable to change passwords.

SQLyog 13.1.4 (64-bit)

**Comment by peterwy** [comment]
Verified! Error seen in attached image.  
![alteruser](https://user-images.githubusercontent.com/11457737/60720800-28005a80-9f2c-11e9-9ac8-5881bd736701.PNG)


On MariaDB 10.1 we cannot use ALTER USER statement, but this (as we do with older servers):
GRANT USAGE ON *.* TO 'user'@'host' IDENTIFIED BY 'newpword';  
-- this will change the pw from 'oldpword' to 'newpword'

**Comment by sibwy** [comment]
Fixed in [v13.1.7](https://blog.sqlyog.com/sqlyog-mysql-gui-13-1-7-released/)
